### PR TITLE
Adds  a new vendor, food and some sprite file presets for some kitchen overhaul elements soon.

### PR DESCRIPTION
**Pull request by Laniakeaaaa**
### What's added/new?:

_Sprite Credits to: Dead.OS/Navi_

### **Vendors:**

**White Dragon Food Vendor:**

- A new vending machine that sells oriental food, chopsticks and some other items...

**New Kitchen Utensils (Totally exciting)**

- New food utensils to eat your food with.

**New Food!**

**Some new oriental food, such as:**
_Dangos
Onigiri
Ramen Bowls
etc._

**Urist Sprite Paths for various kitchen stuffs.**
- Set up some basic new sprite paths, I'll be using for the actual slow overhauls of some urist kitchen-y stuff.

### Images:

WhiteDragon
WhiteDragon2

### Stuff to Fix:

- [x] Make chopsticks not literal hellblades. (They can gore limbs incredibly easily.)
- [x] Fix contraband headgear.
- [x] Finalize chopstick code.

### CODE REVIEW

**Comment by Laniakeaaaa** [comment]
Fixes everything, adds the last remaining bits and bobs. Hopefully this'll all work fine and dandy now, good for a review!

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1225,3 +1225,30 @@
 					/obj/item/weapon/reagent_containers/food/snacks/old/hotdog = 1,
 					/obj/item/weapon/reagent_containers/food/snacks/old/taco = 1
 					)
+
+/obj/machinery/vending/whitedragon
```
Comment (Glloyd) [inline code/game/machinery/vending.dm]: move this code to code/modules/urist/machinery/vending.dm

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -1225,3 +1225,30 @@
...
+	name = "White Dragon"
+	desc = "A vendor shining with bright lamps, it appears to be selling oriental food."
```
Comment (Glloyd) [inline code/game/machinery/vending.dm]: Oriental is generally considered a slur, at least where I'm from. I'd say switch it to Asian, although right now it only seems to be Japanese food, so maybe just change it to that?

**Comment by Glloyd** [comment]
I can fix the icon conflicts for you here and in the other PRs. Icon conflicts are annoying and you have to deal with them manually

File: code/game/machinery/vending.dm (hunk 2 above)
Comment (Laniakeaaaa) [reply code/game/machinery/vending.dm]: Yikes, unintentional. Ported it from a PR I made on an old dead server, I'll change it to that.

**Comment by Laniakeaaaa** [comment]
Made relevant changes

**Pull request merged**